connectors/src/api/add_webhook_router_config.ts

```diff
@@ -0,0 +1,191 @@
+import type { Request, Response } from "express";
+import { isLeft } from "fp-ts/lib/Either";
+import * as t from "io-ts";
+import * as reporter from "io-ts-reporters";
+
+import { connectorsConfig } from "@connectors/connectors/shared/config";
+import { NotionConnectorState } from "@connectors/lib/models/notion";
+import {
+  SigningSecretMismatchError,
+  WebhookRouterConfigService,
+} from "@connectors/lib/webhook_router_config";
+import logger from "@connectors/logger/logger";
+import { apiError, withLogging } from "@connectors/logger/withlogging";
+import { SlackConfigurationResource } from "@connectors/resources/slack_configuration_resource";
+import type { WithConnectorsAPIErrorReponse } from "@connectors/types";
+
+type WebhookRouterEntryParams = {
+  provider: "slack" | "notion";
+  providerWorkspaceId: string;
+};
+
+type WebhookRouterEntryResBody = WithConnectorsAPIErrorReponse<{
+  success: boolean;
+}>;
+
+const AddWebhookRouterEntryBodySchema = t.intersection([
```

Comment:
No strong opinion, but I heard we wanted to migrate from io-ts validation to zod validation ?